README.md

```diff
@@ -1,17 +1,74 @@
-[![CircleCI](https://dl.circleci.com/status-badge/img/gh/giantswarm/APP-NAME/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/giantswarm/APP-NAME/tree/main)
+[![CircleCI](https://dl.circleci.com/status-badge/img/gh/giantswarm/kyverno-policy-operator/tree/main.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/giantswarm/kyverno-policy-operator/tree/main)
 
 [Read me after cloning this template (GS staff only)](https://handbook.giantswarm.io/docs/dev-and-releng/app-developer-processes/adding_app_to_appcatalog/)
 
-# {APP-NAME} chart
+# kyverno-policy-operator chart
 
-Giant Swarm offers a {APP-NAME} App which can be installed in workload clusters.
-Here we define the {APP-NAME} chart with its templates and default configuration.
+Giant Swarm offers a kyverno-policy-operator App which can be installed in workload clusters.
+Here we define the kyverno-policy-operator chart with its templates and default configuration.
 
-**What is this app?**
+Kyverno Policy Operator reconciles the GiantSwarm PolicyException instances and creates the necessary Kyverno PolicyExceptions objects.
 
-**Why did we add it?**
+A GiantSwarm PolicyException consists of a list of Policies and Targets to be excluded from the Kyverno Policy Engine. Having a GiantSwarm PolicyException will ensure that workloads targeted by that policy won't be blocked during Admission time by Kyverno.
 
-**Who can use it?**
+### Sample GiantSwarm PolicyException:
+
+In some cases, it may be necessary to exempt specific resources from the enforcement of Kyverno policies, such as `disallow-privilege-escalation` and `require-run-as-nonroot`. To achieve this, you can create a GiantSwarm PolicyException. Below is an example of how to exclude the `my-custom-operator` Deployment in the `default` namespace from these policies:
```

Comment:
```suggestion
In some cases, it may be necessary to exempt specific resources from the enforcement of Kyverno policies, such as `disallow-privilege-escalation` and `require-run-as-nonroot`. To achieve this, you can create a Giant Swarm PolicyException. Below is an example of how to exclude the `my-custom-operator` Deployment in the `default` namespace from these policies:
```